src/apps/backend/modules/config/config.py

```diff
@@ -0,0 +1,157 @@
+import os
+from pathlib import Path
+from typing import Any, Optional
+import yaml
+
+
+def get_parent_directory(directory: str, levels: int) -> Path:
+    parent_dir = Path(directory)
+    for _ in range(levels):
+        parent_dir = parent_dir.parent
+    return parent_dir
+
+
+class Config:
```

Comment:
If this is not supposed to be file consumed by other modules directly, this should live inside `internals` folder